app/containers/HomeContainer/index.js

```diff
@@ -56,6 +63,8 @@ export function HomeContainer({
   useInjectReducer({ key: 'homeContainer', reducer });
   useInjectSaga({ key: 'homeContainer', saga });
   const [loading, setLoading] = useState(false);
+  const [text, setText] = useState('');
```

Comment:
why do we need these 2 states?

Reply:
this state use for store searching text and use when user press on the button.